build.gradle

```diff
@@ -24,6 +25,14 @@ repositories {
 	maven { url 'https://repo.spring.io/snapshot' }
 }
 
+docker {
```

Comment:
I feel you can just modify your build.gradle to generate jar file instead line 28 to 35 ??

Reply:
I have updated the code